src/protocols/secure_channel/CASEServer.cpp

```diff
@@ -181,4 +180,29 @@ void CASEServer::OnSessionEstablished(const SessionHandle & session)
                     ChipLogValueScopedNodeId(session->GetPeer()));
     PrepareForSessionEstablishment(session->GetPeer());
 }
+
+void CASEServer::SendBusyStatusReport(Messaging::ExchangeContext * ec)
+{
+    using namespace Protocols::SecureChannel;
+
+    ChipLogProgress(SecureChannel, "Already in the middle of CASE handshake, sending busy status report");
+
+    // A successful CASE handshake can take several seconds and some may time out (30 seconds or more).
+    // TODO: Come up with better estimate: https://github.com/project-chip/connectedhomeip/issues/28288
+    // For now, setting minimum wait time to 5 seconds.
+    uint16_t minimumWaitTime                            = 5 * 1000;                // milliseconds
```

Comment:
The time here MUST be an argument to SendBusyStatusReport, so that it can be updated.

Reply:
+1. Passing the time to this function gives better flexibility

Reply:
Added it as an argument.